http/dotnet/httpclient/src/HttpClientRequestAdapter.cs

```diff
@@ -173,9 +181,10 @@ public async Task<ModelType> SendPrimitiveAsync<ModelType>(RequestInformation re
         /// </summary>
         /// <param name="requestInfo">The <see cref="RequestInformation"/> instance to send</param>
         /// <param name="responseHandler">The <see cref="IResponseHandler"/> to use with the response</param>
+        /// <param name="errorMapping">The error factories mapping to use in case of a failed request.</param>
         /// <param name="cancellationToken">The <see cref="CancellationToken"/> to use for cancelling the request.</param>
         /// <returns></returns>
-        public async Task SendNoContentAsync(RequestInformation requestInfo, IResponseHandler responseHandler = null, CancellationToken cancellationToken = default)
+        public async Task SendNoContentAsync(RequestInformation requestInfo, IResponseHandler responseHandler = null, Dictionary<string, Func<IParsable>> errorMapping = default, CancellationToken cancellationToken = default)
```

Comment:
We missed a spot in this function. It is missing this call.
```cs
await ThrowFailedResponse(response, errorMapping);
```

Reply:
thanks for the call out, fixed